docs_src/docs/api_guide/components/networking/networking.md

```diff
@@ -116,7 +116,10 @@ Ethernet Low-Level Driver (\ref ENET_LLD) is a driver that aims at providing an
 
 
 ### TCP/IP Stack
--\subpage NETWORKING_LWIP
```

Comment:
we shouldn't remove any existing section. Any reason why LWIP section is removed ?